src/routes/settings/containers/Settings.js

```diff
@@ -0,0 +1,23 @@
+const Connect = require('react-redux').connect;
+const Settings = require('../components/Settings');
+// const AuthAct = require('actions/auth');
+const AuthSelectors = require('selectors/auth');
+
+const internals = {};
+
+internals.connect = Connect(
+    (state) => ({
+        errorMessage: state.auth.error.message,
+        userDetails: {
+            firstName: AuthSelectors.getUserName(state),
+            lastName: AuthSelectors.getUserLastName(state),
+            email: AuthSelectors.getUserEmail(state)
+        }
+    }),
+    {
+        // onSubmit: AuthAct.registerUser
+        onSubmit: (console.log('Trying to submit on Settings page'))
```

Comment:
This looks like it will only run once when the app initializes.  Do you want `() => console.log(/* ... */)` ?